### DISCUSSION TITLE
ANKI PLUGIN

### DISCUSSION THREAD

**Discussion by m1e2d3** [TriliumNext/Trilium#2729] [Ideas]
Can we have a anki plug-in for trillium  ?

**Comment by MrOctocat** [comment]
Make a new note, change the note type to Web View (which is now stable and works just fine in Trilium), add the attribute `#webViewSrc="https://ankiweb.net/decks" #iconClass="bx bxs-card"` to your note, press Ctrl+R to reload the frontend, log into the page that appears with your AnkiWeb credentials, and there you go. Not only do you have access to your cards to study via the Trilium interface, but you can also modify them from there too. This will avoid you having to constantly switch from Trilium to your browser or Anki app, if that was the intention behind that plugin request.

Unfortunately, the Anki website doesn't generate a unique link for each individual deck, so you can't really have them in separate notes, ready to study. You have to go through the home decks page each time. Hopefully this might change in the future, or perhaps there is already a trick for this and I'm not aware of it.

Do note that the Anki app remains much more powerful than the web interface, so exploiting those features will require to make the little sacrifice of switching windows when you want to study your flashcards. (Of course, split screen is also a thing.)

If what you meant by an "Anki plugin" is having functionality between Trilium and Anki, such as links to specific decks or cards from within notes, that's not really possible, given that Anki doesn't have an API dedicated to that sort of thing. In other words, that's on Anki developers, and not Zadam & Contributors.

**Comment by vultankingofthehawkmen** [comment]
Obsidian has a plugin that allows anki cards to be defined inside notes. the knowledge base can then be scanned for card definitions and exported -> imported to anki. This proves to be a really useful workflow as notes and their related cards will be maintained in the same place at the same time.
Just clarifying what people are generally looking for when asking for an anki plugin

**Reply by WantToLearnJapanese** [reply]
Interesting.
By the way, I think we need more people for the Trilium community.
We all know that everything is possible with Trilium, but the number of custom plugins and its popularity are amazingly tiny.

**Comment by Gib-git** [comment]
made this, see if its useful for you guys

https://github.com/Gib-git/trilium-ai-anki-addon

demo:
